docs/modules/eprescription/prescribing/rx-change.md

```diff
@@ -0,0 +1,96 @@
+# RxChange Message
+
+### Overview
+
+RxChange is a type of message sent by pharmacy to subscriber when the medication should be changed to a different one.
+It is required that RxChange is preceded by NewRx.
+
+### Receiving RxChangeRequest
+
+Once the pharmacy sends an RxChangeRequest, ePrescription module receives Surescripts message at `/eprescriptions/rx` endpoint and saves it to Aidbox.
+Resources created:
+
+- `Provenance` - solely as a record that an event has occurred.
+  It is intended to serve as an audit/logging artifact, and should not be relied upon for driving business logic.
+- `MedicationRequest` reflecting the incoming data
+- `DetectedIssue` - if only there were some mismatches in related resources
+
+There are several ways to track the newly created **MedicationRequest**s:
+
+- [Either of Aidbox Subscriptions mechanisms](../../topic-based-subscriptions/README.md)
+- Manual/automated tracking based on **MedicationRequest** modification date and/or status
+
+The ePrescription module creates a `DetectedIssue` whenever inbound RxRenewal data diverges from existing records; see [Detected Issues](./detected-issue.md) for the full list of checks and a sample payload.
+
+### Supported Request Types
+
+- **D**: Drug Use Evaluation
+- **G**:  Generic Substitution - A modification of the product prescribed to a generic equivalent.
+- **OS**: Pharmacy is out of stock of the medication prescribed and it cannot be obtained in a clinically appropriate timeframe.
+<!-- - **P**: Prior Authorization Required - A request to obtain prior authorization before dispensing. -->
```

Comment:
Should we keep it as comment, or include it with a note on lack of support? I'd leave it as a comment, but I trust your judgement.